openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -37,32 +37,32 @@ def _make_profile_image_name(username):
     return hashlib.md5(settings.PROFILE_IMAGE_SECRET_KEY + username).hexdigest()
 
 
-def _get_profile_image_filename(name, size):
+def _get_profile_image_filename(name, size, version):
     """
     Returns the full filename for a profile image, given the name and size.
     """
-    return '{name}_{size}.jpg'.format(name=name, size=size)
+    return '{name}_{size}.{version}.jpg'.format(name=name, size=size, version=version)
```

Comment:
Got it. I'm cool with the dot, by the way. It looks like Google [recommends it as well](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#invalidating-and-updating-cached-responses).